prowler/providers/aws/aws_provider.py

```diff
@@ -462,9 +470,12 @@ def setup_session(
                 )
         except Exception as error:
             logger.critical(
-                f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+                f"AWSSetUpSessionError[{error.__traceback__.tb_lineno}]: {error}"
+            )
+            raise AWSSetUpSessionError(
+                original_exception=error,
+                file=pathlib.Path(__file__).name,
```

Comment:
Do you still need this?

Reply:
Yes, i raise this error to manage it.
An example can be found inside `test_connection`